tailcfg/tailcfg.go

```diff
@@ -1579,6 +1579,9 @@ type SSHRule struct {
 	// Action is the outcome to task.
 	// A nil or invalid action means to deny.
 	Action *SSHAction `json:"action"`
+
+	// Targets matches an incoming connection for the target destination.
```

Comment:
1) this should go before the Action field.

but:

2) why would we sent a SSHPolicy down to a node talking about Targets under than the node we sent the policy too? The only Target that is applicable is the current node itself.